content/trailmap/prologue.md

```diff
@@ -0,0 +1,159 @@
+date = "2021-12-10"
+type = "about"
+[ author ]
+name = "Dogecoin Foundation"
++++
+
+<section class="presentation">
+<div class="left">
+
+<div class="title">
+
+
+ ## <img title='You are here!' width="60px" style='display: inline;' src="/marker.png"/>Dogecoin Trailmap: Prologue
+
+<div class="underline"></div>
+</div>
+
+<div class="description">
+ 
+* [**Dogecoin Trailmap: Prologue**](/trailmap/prologue/) 
+* [Project: Libdogecoin](/trailmap/libdogecoin/)
+* [Project: Dogecoin Standard](/trailmap/standard/)
+* [Project: GigaWallet](/trailmap/gigawallet/)
+* Project: Dogecoin Keyring (App / SDK)
+* Task: Doge-In-A-Box (Core Containerisation)
+* Project: RadioDoge (Starlink + Radio nodes for remote locations)
+* Proposal: Community Staking (Proof of Stake)
+* Partnership: Point of Sale (L2 app) 
+</div>
+
+</div>
+
+<div class="right">
+<img class="dogegoin-light" src="/trailmap.png" alt="Dogecoin logo">
+<img class="dogegoin-dark" src="/trailmap.png" alt="Dogecoin logo">
+</div>
+
+
+</section>
+
+<section class='board'>
+
+## What is the Dogecoin Trailmap?
+
+While _roadmap_ is a term that's commonly used when new cryptocurrencies 
+are released, it conveys a rigid way forward, led by a single company 
+or _'lead'_.  
+
+**Dogecoin is a community-driven, consensus based open-source project and its 
+direction is formed by the many individual and organisational contributors 
+to the various projects of the Dogecoin blockchain & ecosystem.**  
+
+The direction of Dogecoin is paved by the many projects that the community 
+feels best shapes its direction. This is a very intentional decision: it is 
+important that Dogecoin remains a decentralised project with shepherds rather 
+than rulers, each whose projects and contributions stand or fall on their 
+merits.
+
+This Trailmap represents a dog-walk the Dogecoin Foundation is committed
+to exploring and which it believes are beneficial for the success of Dogecoin.
+
+
+## Why these projects? 
+
+Back in August we signed the [Dogecoin Manifesto](/manifesto) and welcomed 
+the community to sign it by clicking the tweet button; the response was 
+overwhelmingly positive.  
+
+The Manifesto was our attempt at reducing down everything the community wanted 
+from the Dogecoin project: a 'Currency of the people, for the people and by the 
+people', something humanity could really use, to buy a coffee or pay the rent.
+
+Therefore the first pillar of the Manifesto is: **Being useful, we value utility 
+over technical brilliance.**
+
+This is fundamental to the thinking behind the Foundation Trailmap, in fact we 
+believe this so strongly we have an even simpler saying:
+</br>utility → adoption,
+</br>utility → adoption!
+</br>utility → adoption.. this is the way.
+
+Essentially the more utility the more useful, the more useful the more adoption, 
+the more adoption the more Dogecoin gains traction as a serious universal currency 
+for people everywhere. 
+
+## Two coins starting with D..
```

Comment:
there are two '.' here, is it a typo issue?

Reply:
That's a two-dot ellipsis :)   I'm not sure how that would get translated?

Reply:
changed it to 3 dots